data_collection/gazette/spiders/sp_sao_sebastiao.py

```diff
@@ -0,0 +1,51 @@
+import datetime as dt
```

Comment:
```suggestion
from datetime import date, datetime
```
Dessa forma o código fica mais legível e segue os padrões de outros imports no projeto.